sdcm/tester.py

```diff
@@ -974,21 +973,24 @@ def set_system_auth_rf(self, db_cluster=None):
         if self.test_config.REUSE_CLUSTER:
             self.log.debug("Skipping change rf of system_auth for reusing cluster")
             return
-        node = self.db_cluster.nodes[0]
+        self.set_ks_rf(keyspace="system_auth", db_cluster=db_cluster)
+
+    def set_ks_rf(self, keyspace, db_cluster=None):
```

Comment:
Name is a bit vague.

It doesn't state to which replication factor it's gonna change and it's not in the parameters.

Also seems like this function belongs to Cluster and not Tester

Reply:
> Name is a bit vague.

> It doesn't state to which replication factor it's gonna change and it's not in the parameters.

I don't disagree, I was just compiling with the previously written method